superbench/runner/runner.py

```diff
@@ -417,6 +429,8 @@ def run(self):
             if benchmark_name not in self._sb_enabled_benchmarks:
                 continue
             benchmark_config = self._sb_benchmarks[benchmark_name]
+            if benchmark_name == 'ib-traffic' and 'topo-aware' in benchmark_config.parameters.values():
+                self.generate_ibstat()
```

Comment:
@abuccts any idea on this? It's strange to change the general-purpose code for one specific testing? In future, the code is hard to maintain

Reply:
If we think it's basic information task, maybe we should add a command line for this

Reply:
@RyoYang  Instead of here, is it possible to add this functionality into ib_validation_performance.py (when user specifies topo-aware, and not providing ibstat output in the config)?

Reply:
Actually, I used to add this part into topo-aware module, you can see the changes in this branch (https://github.com/RyoYang/superbenchmark/commits/ibstat-auto-generation), but there has some concurrency issues when generate the ibstat file, as each node will generate a file for others.